src/libraries/System.Private.Xml/src/System/Xml/Schema/XsdBuilder.cs

```diff
@@ -778,7 +778,7 @@ internal override bool IsContentParsed()
             return _currentEntry.ParseContent;
         }
 
-        internal override void ProcessMarkup(XmlNode[] markup)
+        internal override void ProcessMarkup(XmlNode?[] markup)
```

Comment:
Why can't markup be null here?

Reply:
`markup` is always initialized in the only call site.
https://github.com/dotnet/runtime/blob/454d66db477ea269e729cfb415eb615cc102ea87/src/libraries/System.Private.Xml/src/System/Xml/Schema/Parser.cs#L246-L252